.circleci/config.yml

```diff
@@ -81,6 +81,30 @@ jobs:
       - run:
           name: Publish package
           command: "./node_modules/.bin/lerna publish --message 'chore: release new versions' --yes"
+  generate-repo:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/
+      - add_ssh_keys:
+          fingerprints:
+            - '95:74:63:db:82:da:b3:e6:57:0c:3a:bd:3f:ff:ce:c6'
```

Comment:
This is my user key...we should probably have a Machine Key for the team.

Reply:
Yeah.. I have no idea how this works though :(

Reply:
IIUC, we just create a new GitHub account `open-wc-machine-user` (or better, damn we've got to name another thing), and then give it specific access/an SSH key and then Bob's you're uncle...